Install/Chroot

```diff
@@ -268,7 +268,30 @@ Now you're ready to boot into Source Mage GNU/Linux system, so type:
 
 **9. Continue on**
 
-Follow the [[Install/ISO#post-install|post-installation steps]] in our [[Install/ISO|ISO install guide]] to update and optimize the system for your hardware.
+As an option, you may with to install and view the "smgl-setup" man page:

```

Comment:
wish

Reply:
Another typo. This has been corrected and should be part of the next pull request.